src/components/FeatureViews/Courses/ClassEnrollmentRow.js

```diff
@@ -0,0 +1,363 @@
+    margin: "0px 24px 25px 24px",
+    paddingTop: "0px",
+  },
+  studentRenderAccordionSpacing: {
+    width: "202px",
+  },
+  parentRenderAccordionSpacing: {
+    width: "200px",
+  },
+  actionsRenderAccordionSpacing: {
+    width: "200px",
+  },
+  iconRenderAccordionSpacing: {
+    position: "relative",
+    left: "6.5em",
+  },
+  studentInfoSpacing: {
+    margin: "15px",
+  },
+}));
+
+const ClassEnrollmentRow = ({
+  fullStudentName,
+  accountType,
+  studentId,
+  studentInfo,
+  parentAccountType,
+  parentId,
+  concatFullParentName,
+  phoneNumber,
+  handleOpenModal,
+  enrollmentList,
+  enrollmentID,
+  courseTitle
+}) => {
+  const { location } = useHistory();
+  const paramsID = useParams();
+  const dispatch = useDispatch();
+
+  let courseID;
+
+  paramsID.id ? (courseID = paramsID.id) : (courseID = paramsID.courseID);
+
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [expanded, setExpanded] = React.useState(false);
+  const [studentMenuAnchorEl, setStudentMenuAnchorEl] = useState(null);
+  const [unenroll, setUnenroll] = useState({
+    enrollment: null,
+    open: false,
+  });
+
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+
+  const handleClose = () => setAnchorEl(null);
+
+  const handleOpen = (e) => {
+    e.preventDefault();
+    const currentValue = e.currentTarget.value;
+    const dataType = e.currentTarget.dataset.type;
+    handleOpenModal(currentValue, dataType);
+    setAnchorEl(null);
+  };
+
+  const handleChange = () => {
+    setExpanded(!expanded);
+  };
+
+  const handleClickStudentMenu = useCallback(({ currentTarget }) => {
+    setStudentMenuAnchorEl(currentTarget);
+  }, []);
+
+  const handleCloseStudentMenu = useCallback(() => {
+    setStudentMenuAnchorEl(null);
+  }, []);
+
+  const handleUnenroll = useCallback(
+    (enrollment) => () => {
+      setUnenroll({
+        enrollment,
+        open: true,
+      });
+    },
+    []
+  );
+
+  const closeUnenrollDialog = useCallback(
+    (toUnenroll) => () => {
+      if (toUnenroll) {
+        deleteEnrollment(unenroll.enrollment)(dispatch);
+      }
+      setUnenroll({
+        enrollment: null,
+        open: false,
+      });
+      setStudentMenuAnchorEl(null);
+    },
+    [dispatch, unenroll.enrollment]
+  );
+
+  return (
+    <>
+    <Accordion expanded={expanded} classes={{ root: classes.MuiAccordionroot }}>
+      <AccordionSummary
+        expandIcon={
+          <ExpandMoreIcon
+            style={{ color: omouBlue }}
+            fontSize="large"
+            onClick={handleChange}
+          />
+        }
+        aria-controls={"studentinfo-" + studentId + "-content"}
+        id={"studentinfo-" + studentId + "-details"}
+        eventKey={"studentinfo-" + studentId + "-details"}
+      >
+        <TableRow
+          key={fullStudentName}
+          className={
+            location.pathname === `/coursemanagement/class/${courseID}`
+              ? ""
```

Comment:
you can simplify this with the && operator:
```
(location.pathname === `/coursemanagement/class/${courseID}`) && classes.center
```

Reply:
Updated, added a bang operator since if it is false!